modules/user-guide/pages/collating-vale-output.adoc

```diff
@@ -0,0 +1,70 @@
+// Metadata for Antora
+:navtitle: Collating Vale output
+:keywords: collating, output
+:description: Collating the Vale CLI report results
+:page-aliases: end-user-guide:collating-vale-output.adoc
+// End of metadata for Antor
+
+:context: collating-vale-output
+:_module-type: CONCEPT
+[id="collating-vale-output_{context}"]
+= Collating Vale output
+
+As a technical writer, you can use a template to collate the results from running `vale` on the command line. This collated output is non-repetitive and might be easier to use.
+
+The `RedHat` style includes a `collate-output.tmpl` template you can specify when you run the `vale` command:
+
+----
+$ cd openshift-docs/logging/troubleshooting/
+$ vale --output='~/styles/collate-output.tmpl' cluster-logging-troubleshooting-for-critical-alerts.adoc
```

Comment:
```suggestion
$ vale --output='./.vale/styles/RedHat/collate-output.tmpl' logging/troubleshooting/cluster-logging-troubleshooting-for-critical-alerts.adoc
```

Reply:
The idea here is that we `vale sync` in `~/openshift-docs`, and then run Vale from the root project dir, this allows us to use the template using a relative path.

Reply:
depending on how you config Vale I guess

Reply:
Updated. Thanks.

Reply:
I'm not sure this works. Let's talk about it next week.